docs/Getting Started/B - Benefits.md

```diff
@@ -0,0 +1,8 @@
+# Benefits
+
+Finch's Benefits endpoints allows users to read and write benefits information under a single, unified API. The table below lays out which benefits features are supported for providers, along with any notable information.
+
+
+ Provider | Read Company Benefits | Create Company Benefits | Read Individual Benefits | Enroll/Unenroll Individual Benefits | Notes
+---------|----------|---------
+ Gusto | 401(k), Roth 401(k), 403(b), Roth 403(b), Simple IRA, HSA (post tax), FSA Dependent, FSA Medical, S125 Medical/Dental/Vision, Commuter Pre Tax, Custom Post Tax | 401(k), Roth 401(k), 403(b), Roth 403(b), Simple IRA, HSA (post tax), FSA Dependent, FSA Medical, S125 Medical/Dental/Vision, Commuter Pre Tax, Custom Post Tax | 401(k), Roth 401(k), 403(b), Roth 403(b), Simple IRA, HSA (post tax), FSA Dependent, FSA Medical, S125 Medical/Dental/Vision, Commuter Pre Tax, Custom Post Tax | 401(k), Roth 401(k), 403(b), Roth 403(b), Simple IRA, HSA (post tax), FSA Dependent, FSA Medical, S125 Medical/Dental/Vision, Commuter Pre Tax, Custom Post Tax | Percentage-based custom post-tax recurring deductions are not currently supported.
```

Comment:
is there anywhere we can state one-time post tax deductions aren't supported either?